test/testmetrics/metrics/04-teststep-retrieve-metrics-via-localhost.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1
+kind: TestStep
+commands:
+  - script: ./retrieve-metrics2.sh "keptn_lifecycle"
```

Comment:
Please rename the script appropiately, also pass the whole metric name as an argument, not only the  component name